ecosystem/python/sdk/examples/multisig.py

```diff
@@ -0,0 +1,468 @@
+import subprocess
+import time
+
+from aptos_sdk.account import Account, RotationProofChallenge
+from aptos_sdk.account_address import AccountAddress
+from aptos_sdk.authenticator import Authenticator, MultiEd25519Authenticator
+from aptos_sdk.bcs import Serializer
+from aptos_sdk.client import FaucetClient, RestClient
+from aptos_sdk.ed25519 import MultiEd25519PublicKey, MultiEd25519Signature
+from aptos_sdk.transactions import (EntryFunction, RawTransaction, Script,
+                                    ScriptArgument, SignedTransaction,
+                                    TransactionArgument, TransactionPayload)
+from aptos_sdk.type_tag import StructTag, TypeTag
+from common import FAUCET_URL, NODE_URL
+
+
+def wait():
+    """Wait for user to press Enter before starting next section."""
+    input("\nPress Enter to continue...")
+
+
+if __name__ == "__main__":
+    rest_client = RestClient(NODE_URL)
+    faucet_client = FaucetClient(FAUCET_URL, rest_client)
+
+    # :!:>section_1
+    alice = Account.generate()
+    bob = Account.generate()
+    chad = Account.generate()
+
+    print("\n=== Account addresses ===")
+    print(f"Alice: {alice.address()}")
+    print(f"Bob:   {bob.address()}")
+    print(f"Chad:  {chad.address()}")
+
+    print("\n=== Authentication keys ===")
+    print(f"Alice: {alice.auth_key()}")
+    print(f"Bob:   {bob.auth_key()}")
+    print(f"Chad:  {chad.auth_key()}")
+
+    print("\n=== Public keys ===")
+    print(f"Alice: {alice.public_key()}")
+    print(f"Bob:   {bob.public_key()}")
+    print(f"Chad:  {chad.public_key()}")  # <:!:section_1
+
+    wait()
```

Comment:
what's the point of these waits? we want these to run in a test environment... we need to quickly follow up this PR with something that validates this on every PR -- these quickly bitrot or start failing due to other ninjas